spring-boot-h2/spring-boot-h2-database/src/main/java/com/baeldung/h2db/demo/SpringBootApp.java

```diff
@@ -0,0 +1,37 @@
+package com.baeldung.h2db.demo;
+
+import java.sql.SQLException;
+import javax.annotation.PostConstruct;
+import org.h2.tools.Server;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+@SpringBootApplication
+public class SpringBootApp {
+
```

Comment:
use the standard formatter (if using Eclipse) to ensure the indentation meets the requirements as stated in the guidelines (for example, use spaces, not tabs)

Reply:
I am using Intelli J with Google Code Style

Reply:
It's definitely similar to the style we're using. There is a standard formatter available at https://github.com/eugenp/tutorials/tree/master/intelliJ that you can try.
If you can't get that to work, then the main difference I see is in the xml files, where the standard indentation is four spaces per level, and this one seems to be using two spaces.